### PR TITLE
Create may-ide-updates.md

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?

Creates release note for IDE changes. 

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: "May IDE updates and fixes"
+id: "may-ide-updates"
+description: "May 2023 release note: We've ."
+sidebar_label: "Update and fixes: IDE"
+sidebar_position: 7
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]:
```suggestion
sidebar_position: 2
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+- Lint via SQLFluff in beta (reach out if interested, otherwise this will be available for multi-tenant soon!)
+- Format markdown files with prettier
+- Leverage developer experience shortcuts, including `Ctrl + `` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```
Comment (bossnunta) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]: the CTRL + ` is funky looking

**Review by bossnunta — CHANGES_REQUESTED** [review]
can you also help add the BUGS section from the original post in as well

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+sidebar_position: 2
+tags: [May-2023, IDE]
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+- Lint via SQLFluff in beta (reach out if interested, otherwise this will be available for multi-tenant soon!)
```
Comment (bossnunta) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]: let's just say "Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)" and remove the comment about reach out

Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]:
```suggestion
- Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+- Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)
+- Format markdown files with prettier
+- Leverage developer experience shortcuts, including `Ctrl + `` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]:
```suggestion
- Leverage developer experience shortcuts, including `Ctrl + /`` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+- Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)
+- Format markdown files with prettier
+- Leverage developer experience shortcuts, including `Ctrl + /`` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
+- Display parent folder name for files with same name in Changes section
+- Navigate the new IDE features quickly using [the IDE User Interface](https://docs.getdbt.com/docs/cloud/dbt-cloud-ide/ide-user-interface) help page
+- Use `top X` in SQL when previewing in the IDE
+- Opt into the new IDE backend layer over the past month (still with dbt-rpc). Ready for beta later in June!
+
+
+## Product refinements
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md:24]:
```suggestion
## Product refinements 

```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+- Performance-related upgrades:
+    - Reduced cold start time by 60+%
+    - Improved render time of modals in the IDE by 98%
+    - Improved IDE performance with dbt Core v1.5+ (faster and snappier – highly encourage you to update your dbt version!)
+- Upgraded sqlfmt (which powers the Format button) to 0.18.0
+- Updated Build button to change menu options based on file/model type (snapshot, macro, etc.)
+- Display message to disable adblocker for file contents error
+- Moved Format button to console bar
+- Made many security enhancements in the IDE
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md:34]:
```suggestion
- Made many security enhancements in the IDE
## Bug fixes

- File icon sizes no longer get wonky in small screen
- Toast notifications no longer take over command bar menu
- Hover info inside the text editor no longer gets cut off
- Transition between a file and a recently modified scratchpad no longer triggers a console error
- dbt v1.5+ now can access the IDE
- Confirm button on the Unsaved Changes modal now closes after clicking it
- Long node names no longer overflow in the parsed logs section in history drawer
- Status pill in history drawer no longer scales with longer command
- Tooltip for tab name with a long file name is no longer cut off
- Lint button should no longer available in main branch
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+- Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)
+- Format markdown files with prettier
+- Leverage developer experience shortcuts, including `Ctrl + /`` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]:
```suggestion
- Leverage developer experience shortcuts, including `Ctrl + \`` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+- Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)
+- Format markdown files with prettier
+- Leverage developer experience shortcuts, including `Ctrl + \`` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]:
```suggestion
- Leverage developer experience shortcuts, including `Ctrl + \`` (toggle history drawer), `CMD + Option + \/` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+- Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)
+- Format markdown files with prettier
+- Leverage developer experience shortcuts, including `Ctrl + \`` (toggle history drawer), `CMD + Option + \/` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]:
```suggestion
- Leverage developer experience shortcuts, including ``Ctrl +` `` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
+---
+title: "May IDE updates and fixes"
+id: "may-ide-updates"
+description: "May 2023 release note: We've ."
+sidebar_label: "Update and fixes: IDE"
+sidebar_position: 2
+tags: [May-2023, IDE]
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]:
```suggestion
To continue improving your [Cloud IDE](/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+id: "may-ide-updates"
+description: "May 2023 release note: We've ."
+sidebar_label: "Update and fixes: IDE"
+sidebar_position: 2
+tags: [May-2023, IDE]
+---
+
+To continue improving your [Cloud IDE](/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]:
```suggestion
Stay up-to-date with [IDE-related changes](/tags/ide).
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
+---
+title: "May IDE updates and fixes"
+id: "may-ide-updates"
+description: "May 2023 release note: We've ."
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]:
```suggestion
description: "May 2023 release note: We've launched SQLFluff in beta, released an IDE UI page, significantly improved IDE performance, improved error messages, fixed bugs, and more."
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+To continue improving your [Cloud IDE](/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Stay up-to-date with [IDE-related changes](/tags/ide).
+
+## New features 
+- Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)
+- Format markdown files with prettier
+- Leverage developer experience shortcuts, including ``Ctrl +` `` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]:
```suggestion
- Leverage developer experience shortcuts, including ``Ctrl + ` `` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+- Use `top X` in SQL when previewing in the IDE
+- Opt into the new IDE backend layer over the past month (still with dbt-rpc). Ready for beta later in June!
+
+
+## Product refinements 
+
+- Performance-related upgrades:
+    - Reduced cold start time by 60+%
+    - Improved render time of modals in the IDE by 98%
+    - Improved IDE performance with dbt Core v1.5+ (faster and snappier – highly encourage you to update your dbt version!)
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]: sugg linking to upgrade page

```suggestion
    - Improved IDE performance with dbt Core v1.5+ (faster and snappier – highly encourage you to [upgrade your dbt version](/docs/dbt-versions/upgrade-core-in-cloud)!)
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 15]
```diff
@@ -0,0 +1,46 @@
+---
+title: "May IDE updates and fixes"
+id: "may-ide-updates"
+description: "May 2023 release note: We've ."
+sidebar_label: "Update and fixes: IDE"
+sidebar_position: 2
+tags: [May-2023, IDE]
+---
+
+To continue improving your [Cloud IDE](/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]:
```suggestion
To continue improving your [Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md [hunk 16]
```diff
@@ -0,0 +1,46 @@
...
+To continue improving your [Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Stay up-to-date with [IDE-related changes](/tags/ide).
+
+## New features 
+- Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)
+- Format markdown files with prettier
+- Leverage developer experience shortcuts, including ``Ctrl +` `` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
+- Display parent folder name for files with same name in Changes section
+- Navigate the new IDE features quickly using [the IDE User Interface](https://docs.getdbt.com/docs/cloud/dbt-cloud-ide/ide-user-interface) help page
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md]:
```suggestion
- Navigate the new IDE features quickly using [the IDE User Interface](/docs/cloud/dbt-cloud-ide/ide-user-interface) help page
```

**Review by bossnunta — APPROVED** [review]
looks good to me!

**Review by bossnunta — APPROVED** [review]
looks good to me!

**Pull request merged**